src/components/Spawn/spawnVolumeModal/getFormSchema.tsx

```diff
@@ -0,0 +1,177 @@
+import { css } from "@emotion/react";
+import { add } from "date-fns";
+import { GetFormSchema } from "components/SpruceForm/types";
+import { ExpirationRow } from "../ExpirationRow";
+import { getDefaultExpiration } from "../utils";
+
+interface Props {
+  maxSpawnableLimit: number;
+  availabilityZones: string[];
+  types: string[];
+  hosts: { id: string; displayName: string }[];
+  disableExpirationCheckbox: boolean;
+  noExpirationCheckboxTooltip: string;
+}
+
+export const getFormSchema = ({
+  maxSpawnableLimit,
+  availabilityZones,
+  types,
+  hosts,
+  disableExpirationCheckbox,
+  noExpirationCheckboxTooltip,
+}: Props): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object" as "object",
+    properties: {
+      requiredVolumeInformation: {
+        type: "object" as "object",
+        title: "Required Volume Information",
+        properties: {
+          volumeSize: {
+            title: "Size (GB)",
+            type: "number" as "number",
+            default: maxSpawnableLimit > 500 ? 500 : maxSpawnableLimit,
+            minimum: 1,
+            maximum: maxSpawnableLimit,
+          },
+          availabilityZone: {
+            title: "Region",
+            type: "string" as "string",
+            default: availabilityZones?.[0] ?? "",
```

Comment:
It'd be cool to use a user's preferred AWS region as the default here. But I'm okay with leaving it out for now.